src/snapshot.ts

```diff
@@ -79,7 +79,7 @@ export class SnapshotService implements Services.ServiceInstance {
     async beforeTest(test: Frameworks.Test) {
         this.#currentFilePath = test.file
         this.#currentTestName = `${test.parent} > ${test.title}`
-        await this.#snapshotClient.startCurrentRun(test.file, test.fullTitle, this.#options)
+        await this.#snapshotClient.setup(test.file, this.#options)
```

Comment:
What's the intention of this change?

Reply:
The client has been updated so I got errors, to be compliant I had to change this. The reason is shared in the description of the PR where you can see the changes that ViTest made @christian-bromann

Reply:
Can we update the version in `package.json` then?

Reply:
It already grabs the latest version with the current package.json but I can do so yes

Reply:
Thank you!